tests/no-internal.js

```diff
@@ -124,6 +139,11 @@ ruleTester.run(
           { message: 'class "Internal" is internal.' },
           { message: 'class "Internal" is internal.' },
           { message: 'method "Public.internalMethod" is internal.' },
+          { message: 'class "Internal" is internal.'},
+          { message: 'class "Internal" is internal.'},
+          { message: 'class "Internal" is internal.'},
+          { message: 'class "Internal" is internal.'},
+          { message: 'class "Internal" is internal.'},
```

Comment:
This file is getting quite large, so maybe it would be a good time to extract the `errors` and `code` properties into reusable variables?

Reply:
ya, I will do that

Reply:
did some extraction of a bunch of reused values.